### PR TITLE
Add 7:16 : rest-operator-last

### PR DESCRIPTION
**Pull request by Yatin-kathuria**
### Hello Maintainers
Add 7:16 : rest-operator-last

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -928,6 +928,23 @@ Other Style Guides
     );
     ```
 
+  <a name="functions--rest-operator-last"></a><a name="7.16"></a>
```
Comment (ljharb) [inline README.md:931]: we don't add new sections with numbered links; those are legacy.

**Review by ljharb — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Yatin-kathuria) [reply README.md:931]: I tried to be consistent with the above points as they had been written. If still their is something wrong then can you help me how can I fixed that.

Comment (Yatin-kathuria) [reply README.md]: done

**Review by ljharb — CHANGES_REQUESTED** [review]
`...` is rest syntax; it's not an operator.

However, this is invalid syntax - it's *impossible* to put the rest operator anywhere but last. Why does this need a guide section?

**Comment by ljharb** [comment]
@Yatin-kathuria
> However, this is invalid syntax - it's impossible to put the rest operator anywhere but last. Why does this need a guide section?

**Comment by Yatin-kathuria** [comment]
> @Yatin-kathuria
> 
> > However, this is invalid syntax - it's impossible to put the rest operator anywhere but last. Why does this need a guide section?

ok, it means we are not showing the wrong syntax. I think we can close this PR then.

**Pull request closed without merging**